src/pages/resources/migration/android/migrate-to-3x.md

```diff
@@ -100,6 +100,48 @@ Some of the APIs available in previous major versions of the Mobile SDK for Andr
 
 The `registerExtension` API for each extension that was deprecated in the 2.x version of the mobile SDK has been removed in the 3.x version of the mobile SDK. See the [Update SDK initialization](#update-sdk-initialization) section for more details.
 
+#### MobileCore
+
+| Removed API | Alternative API |
+| :------------- | :-------------- |
+| MobileCore.registerExtension(Class, ExtensionErrorCallback) | [MobileCore.registerExtensions](../../../home/base/mobile-core/api-reference.md#registerextensions) |
```

Comment:
Can you also all the MobileCore APIs which accept ExtensionErrorCallback as parameter to the extension development section?